.github/workflows/cd-controls.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Files Community
+# Licensed under the MIT License.
+
+# Abstract:
+#  Deploys the Files UI Controls library to NuGet
+#
+# Workflow:
+#  1. Restore and build Files UI Controls
+#  2. Generate a NuGet package and symbols
+#  3. Publish the artifacts to NuGet
+
+name: Files CD (UI Controls)
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    runs-on: windows-latest
```

Comment:
```suggestion
    runs-on: windows-latest
    environment: Deployments
```

Reply:
I would strongly recommend against using the same environment for different things.

Reply:
The app CD workflow and UI controls CD workflow should be in different environments

Reply:
I grouped them together as they are both deployments. We can separate them if there is a good reason to, but I may have to generate new tokens.

Reply:
They will need to be separate as the CD for the app requires approval but the CD for the Controls project won't (as it will add extra hassle)

Reply:
The controls project will require approval as well

Reply:
I still think it makes sense to put them in separate deployment environments as they deploy to separate deployment environments

Reply:
Done